ktor-http/ktor-http-cio/jsAndWasmShared/src/io/ktor/http/cio/MultipartJsAndWasm.kt

```diff
@@ -0,0 +1,14 @@
+package io.ktor.http.cio
+
+import io.ktor.utils.io.ByteReadChannel
+import io.ktor.utils.io.cancel
+import io.ktor.utils.io.discard
+import io.ktor.utils.io.exhausted
+
+/*
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
```

Comment:
Minor. Usually we place the copyright note on the first line in a file.

Reply:
👀